includes/Order/Hooks.php

```
@@ -531,4 +510,49 @@ public function handle_order_notes_for_suborder( $order ) {
+    /**
+     * Sync parent order item stock metadata when sub-order item stock is reduced
+     *
+     * When a sub-order item has its stock reduced, also update the parent order item's
+     * _reduced_stock metadata to keep them in sync.
+     * @param WC_Order_Item_Product $item The sub-order item.
+     * @param array                 $change Change details (product, from, to).
+     * @param WC_Order              $order The sub-order.
+     *
+     * @return void
+     */
+    public function sync_parent_order_item_stock( $item, $change, $order ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$change'. (undefined)

(UnusedFormalParameter)